spec/models/player_spec.rb

```diff
@@ -4,4 +4,27 @@
   describe 'relationships' do
     it { should belong_to :game }
   end
+
+  describe 'instance methods' do
```

Comment:
I wonder if theres a better way to give players and games access to their card names